cmd/snap/cmd_run.go

```diff
@@ -52,6 +53,8 @@ type cmdRun struct {
 	Hook     string `long:"hook" hidden:"yes"`
 	Revision string `short:"r" default:"unset" hidden:"yes"`
 	Shell    bool   `long:"shell" `
+	// FIXME: provide a way to pass options to strace
```

Comment:
How about `--strace-opts`?
```
StraceOpts string `long:"strace-opts"`
```

Don't know if `go-flags` can distinguish the case when something is passed in command line but empty and not passed at all. Then you could have:
```
snap run --strace="-v -x -yy" foo
                   ^^^ strace options
```

Reply:
I like this, I will do a followup on this.